mercury-operator/src/main/java/com/redhat/mercury/operator/utils/ResourceUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.redhat.mercury.operator.utils;
+
+public class ResourceUtils {
+
+    public static final String capitalize(Boolean value) {
+        if (Boolean.TRUE.equals(value)) {
+            return "True";
+        }
+        return "False";
+    }
+
```

Comment:
please add:

    public static String now() {
        return ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT);
    }

Reply:
where is it used?

Reply:
Please use it here condition.setLastTransitionTime(new Date().toString());